### PR TITLE
Remove inaccurate confusing signal box from README.md

### PR DESCRIPTION
**Pull request by janeyx99**
Replaces with HUD

### CODE REVIEW

**Review by kit1980 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -39,16 +39,7 @@ You can reuse your favorite Python packages such as NumPy, SciPy, and Cython to
...
-| Linux (ppc64le) GPU | [![Build Status](https://powerci.osuosl.org/job/pytorch-master-nightly-py3-linux-ppc64le-gpu/badge/icon)](https://powerci.osuosl.org/job/pytorch-master-nightly-py3-linux-ppc64le-gpu/) | <center>—</center> |
-| Linux (aarch64) CPU | [![Build Status](http://openlabtesting.org:15000/badge?project=pytorch%2Fpytorch&job_name=pytorch-arm64-build-daily-master-py37)](https://status.openlabtesting.org/builds/builds?project=pytorch%2Fpytorch&job_name=pytorch-arm64-build-daily-master-py37) | [![Build Status](http://openlabtesting.org:15000/badge?project=pytorch%2Fpytorch&job_name=pytorch-arm64-build-daily-master-py38)](https://status.openlabtesting.org/builds/builds?project=pytorch%2Fpytorch&job_name=pytorch-arm64-build-daily-master-py38) |
-
-See also the [CI HUD at hud.pytorch.org](https://hud.pytorch.org/ci/pytorch/pytorch/master).
+Our trunk health (Continuous Integration signals) can be found at [hud.pytorch.org](https://hud.pytorch.org/ci/pytorch/pytorch/master).
```
Comment (albanD) [inline README.md:42]: From the rendered version this is a bit hard to find. I think we could move it up above the TOC?

Comment (seemethere) [reply README.md:42]: I'd agree with this

Comment (janeyx99) [reply README.md]: Done!

**Review by seemethere — APPROVED** [review]

**Pull request closed without merging**